lib/File/Find/Rule.pm

```diff
@@ -541,9 +558,10 @@ sub in {
     my $code = 'sub {
         (my $path = $File::Find::name)  =~ s#^(?:\./+)+##;
         my @args = ($_, $File::Find::dir, $path);
-        my $maxdepth = $self->{maxdepth};
-        my $mindepth = $self->{mindepth};
-        my $relative = $self->{relative};
+        my $maxdepth  = $self->{maxdepth};
+        my $mindepth  = $self->{mindepth};
+        my $relative  = $self->{relative};
+        my $canonpath = $self->{canonpath};
```

Comment:
done

On Thu, Dec 3, 2015 at 6:13 AM, Richard Clamp <notifications@github.com>
wrote:

> In lib/File/Find/Rule.pm
> <https://github.com/richardc/perl-file-find-rule/pull/2#discussion_r46544173>
> :
>
> > @@ -541,9 +558,10 @@ sub in {
> >      my $code = 'sub {
> >          (my $path = $File::Find::name)  =~ s#^(?:\./+)+##;
> >          my @args = ($_, $File::Find::dir, $path);
> > -        my $maxdepth = $self->{maxdepth};
> > -        my $mindepth = $self->{mindepth};
> > -        my $relative = $self->{relative};
> > +        my $maxdepth  = $self->{maxdepth};
> > +        my $mindepth  = $self->{mindepth};
> > +        my $relative  = $self->{relative};
> > +        my $canonpath = $self->{canonpath};
>
> This 'reindent so everything lines up' isn't how the rest of the code is
> generally laid out (there's one exception on line 16 that I'm sad about)
> because it's vcs poison. So yeah, please put lines 561..563 back.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/richardc/perl-file-find-rule/pull/2/files#r46544173>.
>